tests/acceptance/WPFirstCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class WPFirstCest
```

Comment:
Regarding the Cept comment. Codeception supports a variety of ways to write a test. They use Cept(scenario-based format) and Cest (class based format). When the user generates a test, they specifiy (Cept,Cest,Unit) and that gets added on the end of the file name when they generate the test. I guess it's just a formatting they use. A user can generate a test on the command line with something like: `codecept g:cept Acceptance "TestFeature"`. It will generate the filename and the way the code is structured based on (cept,cest declaration).

 I think that's what the _support classes are for. They providing the scaffolding to create that file and to provide more customization if needed.

Either way, I'll try to narrow down all this into a user friendly format.